ddl/serial_test.go

```diff
@@ -152,6 +152,24 @@ func (s *testSerialSuite) TestPrimaryKey(c *C) {
 	c.Assert(err.Error(), Equals, "[ddl:8200]Unsupported drop primary key when alter-primary-key is false")
 }
 
+func (s *testSerialSuite) TestDropAutoIncrementIndex(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	cfg := config.GetGlobalConfig()
+	newCfg := *cfg
+	orignalAlterPrimaryKey := newCfg.AlterPrimaryKey
+	// Only when disable the AlterPrimaryKey config, this test will make sense.
+	newCfg.AlterPrimaryKey = false
```

Comment:
It seems that the default value is false, could we remove it?

Reply:
reasonable.